src/reference/asciidoc/index-header.adoc

```diff
@@ -3,7 +3,7 @@
 Mark Fisher; Marius Bogoevici; Iwein Fuld; Jonas Partner; Oleg Zhurakousky; Gary Russell; Dave Syer; Josh Long; David Turanski; Gunnar Hillert; Artem Bilan; Amol Nayak; Jay Bryant
 
 ifdef::backend-html5[]
-*{project-version}*
```

Comment:
So, looks like `project-version` has gone.

Please, revise all our docs then to replace this one with a new `revnumber`.

For example right now now I see this:
```
<version>{project-version}</version>
```
Or this `compile "org.springframework.integration:spring-integration-amqp:{project-version}"`

Has to be an actual project version we built these docs against.

Reply:
I added a project-version attribute back into the asciidoctor and asciidoctorPdf tasks. That fixed it.